crates/llvm-context/src/polkavm/context/mod.rs

```diff
@@ -30,6 +30,7 @@ use crate::optimizer::Optimizer;
 use crate::polkavm::r#const::*;
 use crate::polkavm::DebugConfig;
 use crate::polkavm::Dependency;
+//use crate::polkavm_evm_event::log;
```

Comment:
```suggestion
```

I think we can as well remove it or is it related to something needing care in a followup?

Reply:
> or is it related to something needing care in a followup?

It can be removed